controllers/books.py

```diff
@@ -21,6 +21,8 @@
 import json
 import logging
 import datetime
+import importlib
```

Comment:
If you want to add a comment to this PR explaining our use of importlib that would be great.